### ISSUE TITLE
Setup guide incomplete?

### ISSUE THREAD

**Issue by benjmd** [tgulacsi/fronius#1]
Any chance the setup guide can be amended to show how to get this going?

**Comment by tgulacsi** [comment]
As the screen capture at https://raw.githubusercontent.com/tgulacsi/fronius/master/screencapture-192-168-1-99-admincgi-bin-1450003121424.png shows,
you have to log in to your fronius, and then set the 
  * "PUSH SERVICE" to "SolarAPI v1", 
  * "HTTP POST", 
  * then the "Server:Port" to the host:port where your instance of this "froninus" proxy listens, 
  * and the "upload filename" to the path your proxy wants - "/v1/current/inverter-{DATE}{TIME}" for example

**Comment by benjmd** [comment]
Yep, so how does that then get pushed to influx/parsed in an influx-friendly way? I see the go scripts, but where do they go?

**Comment by tgulacsi** [comment]
go get -u github.com/tgulacsi/fronius

if you have Go (golang.org/dl) installed - or download the binary from the releases.
